charts/trino/values.yaml

```diff
@@ -96,6 +96,69 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+  # type: configmap
+  # # Rules file is mounted to /etc/trino/resource-groups
+  # configFile: "resource-groups.json"
+  # resourceGroupsFile:
```

Comment:
What's the use case for specifying `configFile`? Also, why `resourceGroupsFile` is an object, if we don't actually allow/use more than one key?